src/StatusQ/Controls/Validators/StatusMinLengthValidator.qml

```diff
@@ -3,9 +3,16 @@ import StatusQ.Controls 0.1
 StatusValidator {
 
     property int minLength: 0
+    property string fieldName: "value"
```

Comment:
Is this necessary? Wouldn't be a general message using "value" be enough?

Reply:
It was a way to retain existing functionality in desktop where error messages specify the name of the field that has an invalid value. Due to the API that allows customisable error messages (via `error`, now `errorMessage`), I will remove this and update on the desktop side where needed.